test/app/features/escape_manual/data/model/escape_manual_fixtures.dart

```diff
@@ -1,7 +1,19 @@
 import 'package:penhas/app/features/appstate/data/model/quiz_session_model.dart';
+import 'package:penhas/app/features/escape_manual/data/model/escape_manual_local.dart';
 import 'package:penhas/app/features/escape_manual/data/model/escape_manual_remote.dart';
 import 'package:penhas/app/features/escape_manual/domain/entity/escape_manual.dart';
 
+final escapeManualLocalModelsFixture = [
+  EscapeManualTaskLocalModel(
+    id: '1',
+    isDone: true,
+  ),
+  EscapeManualTaskLocalModel(
+    id: '3',
+    isRemoved: true,
+  ),
```

Comment:
Não precisaria ter o `updatedAt` aqui também?

Reply:
`updatedAt` é opcional e com valor padrão nulo, a ideia é apenas atribuir quando o envio para a API for feito com sucesso e esse valor será baseado no horário de envio ao API e utilizado como controle no momento de receber as novas atualizações para remover os já atualizados.